bundle/util_test.go

```diff
@@ -0,0 +1,359 @@
+//
+// Copyright (c) 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package bundle
+
+import (
+	"encoding/base64"
+	"fmt"
+	"testing"
+
+	schema "github.com/lestrrat/go-jsschema"
+	ft "github.com/stretchr/testify/assert"
+	yaml "gopkg.in/yaml.v2"
+	"strings"
+)
+
+var emailPlanParams = []ParameterDescriptor{
+	{
+		Name:        "email_address",
+		Title:       "Email Address",
+		Type:        "enum",
+		Description: "example enum parameter",
+		Enum:        []string{"google@gmail.com", "redhat@redhat.com"},
+		Default:     float64(9001),
```

Comment:
Why is the default value for this enum a `float64(9001)`? Is it supposed to fail?

Reply:
I have no idea. Copy paste from broker code will do that :)